src/main/resources/liquibase/changeSetPort.xml

```diff
@@ -19,5 +19,7 @@
     </changeSet>
     
     <changeSet id="00000000000001_dataPort_restore" author="entando" context="restore" />
+    
+    <include file="port/00000000000002_nx_pagemodels.xml" relativeToChangelogFile="true" />
```

Comment:
> a value should be specified for the new columns for old records.
These new columns all have default values so I think this is covered.

> since current version has not be released yet, I would just modify current sql files without adding a new one
This isn't going to be merged to master right away and might be release after 7.0.0.

IMHO we should always create a migration per PR that touches the database. In this case it hasn't been released but we have development and staging environments that could use a seamless db migration without having to recreate the database.

Reply:
We are not sure if WebUI is going to be released with the current version. This is why I created this PR against the `develop-webui` branch and I am keeping these migrations in a separated file.

Reply:
ah ok, you are right then